js/viewer.js

```diff
@@ -183,8 +183,6 @@ function applyAnchor() {
         anchors[0].scrollIntoView(true);
         if (!local)
           window.scrollBy(0, -46); // 46 is the height of the header of Cyberbotics web page
-        else
-          window.scrollBy(0, 180); // manual adjustment for the off-line version
```

Comment:
I think I understood the issue about this.

It's related to images, if the server is too slow to provide them, the scrolling is taking into account only the text, so the anchors above can be delayed (I can reproduce this systematically on my VM-Windows, and sometimes after clearing the cache on my Mac).
The best solution seems to scroll after the images are actually loaded.